src/hypervisor.c

```diff
@@ -107,6 +108,43 @@ cc_oci_expand_net_device_cmdline(struct cc_oci_config *config, guint index) {
 	return g_strdup("");
 }
 
+/*
+ *
+ *
+ */
+#define DPDK_CMDLINE_OBJ "memory-backend-file,id=dpdkmem,size=2048M,mem-path=/dev/hugepages,share=on,prealloc=on"
```

Comment:
Actually, the size is configured at build time, AFAIK.  From the git logs, I think @jodh-intel has some expertise in this?  I agree -- it should be a parameter, and used in both places.